### ISSUE TITLE
WCAG 4.1.2: Ensure buttons have discernible text (.duration-300)

### ISSUE THREAD

**Issue by azap026** [azap026/smetalabv3#87]
#### Issue
`Ensure buttons have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name))
#### Target application
[Smetalab - Smart Engineering & Management](http://localhost:3000/app/guide/works)
#### Element path
.duration-300
#### Snippet
`<button data-slot="tooltip-trigger" data-variant="outline" data-size="icon" class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium disabled:pointer-events-none disabled:opacity-50 [&amp;_svg]:pointer-events...`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element does not have an implicit (wrapped) <label>
      Element does not have an explicit
